contrib/core-contract-tests/tests/bns/name_register_test.ts

```diff
@@ -98,7 +98,7 @@ Clarinet.test({
                 ], 
                 cases[0].nameOwner.address),
         ]);
-        assertEquals(block.height, 2);
+        assertEquals(block.height, 3);
```

Comment:
This looks like there was an off-by-one error somewhere, with all these changes. Can you elaborate more on why all of these block heights are now +1?

Reply:
The pox-4 is in a different epoch than bns, therefore booting needs one more block